runtimevar/runtimeconfigurator/example_test.go

```diff
@@ -63,5 +63,10 @@ func ExampleNew() {
 	defer v.Close()
 
 	// You can now read the current value of the variable from v.
-	// Snapshot.Value will be of type MyConfig.
+	snapshot, err := v.Watch(context.Background())
+	if err != nil {
+		log.Fatal(err)
+	}
+	// The resulting runtimevar.Snapshot.Value will be of type MyConfig.
```

Comment:
You could write `var val MyConfig = runtimevar.Snapshot.Value` instead of the comment to have the test check the claim.

Reply:
`Snapshot.Value` is in interface{}, so that assignment won't work without a cast.

Reply:
Why not do the cast?

Why not add an `// Output` line?

Reply:
> Why not do the cast?
? I do do the cast, it's on the next line. I don't think it makes it clearer to split it into two lines.

> Why not add an // Output line?
With what? These examples are for non-local providers, so they aren't going to actually work if run. All it could do is print out some static text, or an error.